lib/slave/include/kickcat/AbstractESC.h

```diff
@@ -26,8 +26,8 @@ namespace kickcat
     #define SYNC_MANAGER_PI_IN(index, address, length)  SyncManagerConfig{index, address, length, 0x20, SyncManagerType::Input}
     #define SYNC_MANAGER_PI_OUT(index, address, length) SyncManagerConfig{index, address, length, 0x64, SyncManagerType::Output}
 
-    #define SYNC_MANAGER_MBX_IN(index, address, length)  SyncManagerConfig{index, address, length, 0x02, SyncManagerType::MailboxIn}
-    #define SYNC_MANAGER_MBX_OUT(index, address, length) SyncManagerConfig{index, address, length, 0x06, SyncManagerType::MailboxOut}
+    #define SYNC_MANAGER_MBX_IN(index, address, length)  SyncManagerConfig{index, address, length, 0x22, SyncManagerType::MailboxIn}
```

Comment:
@leducp  Does this change ok ? This allow the config and the sii to match

Reply:
Removed this change and just testing operation mode and direction in the is_valid_sm()